pysmt/solvers/bdd.py

```diff
@@ -87,6 +148,22 @@ def __init__(self, environment, logic, user_options):
         self.backtrack = []
         self.latest_model = None
 
+    def get_default_options(self, logic=None, user_options=None):
+        res = BddOptions()
+        if user_options is not None:
+            if "unsat_cores_mode" in user_options:
+                if user_options["unsat_cores_mode"] != None:
+                    raise NotImplementedError("BddSolver does not "\
+                                              "support unsat cores")
+
+            for k in ["static_ordering",
```

Comment:
Maybe you can post-pone the initialization of BddOptions till here and do the unpacking of keywords arguments? If they are not defined, you will take the default
```
 BddOptions(**user_options)
```
I think it is nicer and less error-prone than *setattr*